runtests.py

```diff
@@ -29,6 +29,19 @@
 
 ROOT_URLCONF = 'runtests'
 
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [ os.path.join(os.path.dirname(__file__), 'sekizai', 'test_templates'), ],
+        'OPTIONS': {
+            'context_processors': [
+                'sekizai.context_processors.sekizai',
+            ],
+            'debug': True,
```

Comment:
Thank you for this patch. However, could you do me a favour and apply one small change: Use the values of the `TEMPLATE_*` vars in this . So for example `'DIRS': TEMPLATE_DIRS` etc to re-use those values.